diff --git a/mozilla/nsprpub/pr/include/prlog.h b/mozilla/nsprpub/pr/include/prlog.h
index c5f6c62..d2ca5d7 100644
--- a/mozilla/nsprpub/pr/include/prlog.h
+++ b/mozilla/nsprpub/pr/include/prlog.h
@@ -248,14 +248,6 @@ NSPR_API(void) PR_Assert(const char *s, const char *file, PRIntn ln);
 
 #endif /* defined(DEBUG) || defined(FORCE_PR_ASSERT) */
 
-/*
-** Compile-time assert. "condition" must be a constant expression.
-** The macro can be used only in places where an "extern" declaration is
-** allowed.
-*/
-#define PR_STATIC_ASSERT(condition) \
-    extern void pr_static_assert(int arg[(condition) ? 1 : -1])
-
 PR_END_EXTERN_C
 
 #endif /* prlog_h___ */
diff --git a/mozilla/nsprpub/pr/include/prtypes.h b/mozilla/nsprpub/pr/include/prtypes.h
index dd66d0f..fcebc44 100644
--- a/mozilla/nsprpub/pr/include/prtypes.h
+++ b/mozilla/nsprpub/pr/include/prtypes.h
@@ -519,6 +519,14 @@ typedef unsigned long PRUword;
 /********* ????????????? End Fix me ?????????????????????????????? *****/
 #endif /* NO_NSPR_10_SUPPORT */
 
+/*
+** Compile-time assert. "condition" must be a constant expression.
+** The macro can be used only in places where an "extern" declaration is
+** allowed.
+*/
+#define PR_STATIC_ASSERT(condition) \
+    extern void pr_static_assert(int arg[(condition) ? 1 : -1])
+
 PR_END_EXTERN_C
 
 #endif /* prtypes_h___ */
