From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 2 Jan 2016 17:56:03 +0100
Subject: Correctly return exit status on failure

---
 nspr/pr/tests/server_test.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/nspr/pr/tests/server_test.c b/nspr/pr/tests/server_test.c
index 5c608cd..fe2bb92 100644
--- a/nspr/pr/tests/server_test.c
+++ b/nspr/pr/tests/server_test.c
@@ -37,6 +37,7 @@
 #define PASS 0
 #define FAIL 1
 static int debug_mode = 0;
+static int failed_already = 0;
 
 static int _iterations = 1000;
 static int _clients = 1;
@@ -90,6 +91,7 @@ static void Test_Result (int result)
 			break;
 		case FAIL:
 			printf ("FAIL\n");
+			failed_already=FAIL;
 			break;
 		default:
 			break;
@@ -606,5 +608,5 @@ int main(int argc, char **argv)
 
     PR_Cleanup();
 
-    return 0;
+    return failed_already;
 }
